app-management/components/AppDetailsPage/AppEditForm.jsx

```diff
@@ -81,6 +83,15 @@ const EditAppForm = ({ handleSubmit }) => (
             simpleValue
           />
         </section>
+        <section className="col-md-8">
+          <Field
+            component={InputField}
+            type="text"
+            name="legacyId"
+            label="Legacy Id &#42;"
```

Comment:
Can we change this label to just "Legacy Id"? The `&#42;` is only needed to denote that the field is required.